src/System.Private.Xml/tests/Xslt/XsltCompiler/XsltCommon.cs

```diff
@@ -332,22 +333,25 @@ public static bool VerifyAssemblyUsingPEVerify(string asmName, DelayedWriteLogge
 
         public static string SearchPath(String fileName)
         {
-            const string ToolsDirectorySuffix = @"bin\NETFX 4.5.1 Tools";
+            var tools64 = @"C:\program Files (x86)\Microsoft SDKs\Windows";
```

Comment:
since we are modifying this code you might consider changing this to use env variables:
- %programfiles(x86)%
- %programfiles%